pkg/virt-v2v/customize/scripts/rhel/run/network_config_util.sh

```
@@ -490,7 +490,7 @@ udev_from_ifquery() {
         target_ip="$1"
         # Loop through all interfaces and check for the given IP address
         ifquery_get -l | while read -r IFNAME; do
-            if ifquery_get $IFNAME | grep -q "$target_ip"; then
+            if ifquery_get $IFNAME | grep -q "$target_ip\b"; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)